.github/workflows/tests.yml

```diff
@@ -3,7 +3,7 @@ on:
   pull_request:
     branches: ["**"]
   merge_group:
-    branches: ["main"]
+    branches: ['main']
```

Comment:
Why? I see other double quotes around? Let's reduce the diff

Reply:
Right this is copy-pasta from trying to understand why some stuff wasn't working (spoilers: not because of this ^^)